examples/tests/sdk_tests/bucket/list.w

```diff
@@ -0,0 +1,25 @@
+bring cloud;
```

Comment:
The files are saved to the snapshots only by their file name, regarding the path... we already have a "table/list" can you please change this file name to "bucket_list" until we'll fix that issue? (#2781)